project-base/storefront/components/Layout/Footer/FooterMenuItem.tsx

```diff
@@ -9,9 +8,7 @@ type FooterMenuItemProps = {
 
 export const FooterMenuItem: FC<FooterMenuItemProps> = ({ items, title }) => (
     <>
-        <Heading className="text-center font-bold uppercase text-white lg:text-left" type="h3">
```

Comment:
Did we lose the `font-bold` class?

Reply:
Nope, this is already set 
![image](https://github.com/shopsys/shopsys/assets/11465991/3c68c76b-f532-4214-9642-10dc6f5b9b0b)